Dangerfile

```diff
@@ -27,5 +27,15 @@ if has_doc_changes && !has_doc_gen_title
   message("Docs are generated by using [Jazzy](https://github.com/realm/jazzy). If you want to contribute, please update [markdown guides](https://github.com/Instagram/IGListKit/tree/master/Guides)")
 end
 
+# Warn if Source files were added or removed but examples are not updated
+added_source_files = !git.added_files.grep(/Source/).empty?
+deleted_source_files = !git.deleted_files.grep(/Source/).empty?
```

Comment:
can we just use `git.modified_files` for `/Source/` instead of checking added/deleted separately?

Reply:
oops. nevermind. we do need to check this explicitly. 👌